.github/workflows/artifact-release.yml

```diff
@@ -6,48 +6,92 @@ on:
     tags:
       - v*
 
+name: CI
+
 jobs:
   build_appimage:
-    name: "Make Linux AppImage Nightly"
-    runs-on: ubuntu-18.04
+    name: "Build on Ubuntu"
+    runs-on: ${{ matrix.config.os }}
+    strategy:
+      matrix:
+        config:
+        - { 
+            name: "Ubuntu GCC",
+            os:   "ubuntu-20.04",
+            cc:   "gcc-10",
```

Comment:
This choice seems rather conservative, if there's no particular reason, maybe go more modern?

Reply:
The Linuxdeploy script target a minimum GLIB based on the OS of building. Therefore, the resulting AppImage is compatible with GLIBs as old as your build OS and up. 20.04 is already quite modern (CentOS users are probably left out in the cold, which is potentially a problem, it's also the desktop OS in many an institute that rools out Linux machines (Scientific Linux, CernOS etc).

So, it's too modern, if anything :P Maybe the clang build could be bumped, use that job for testing of updated tooling?

Reply:
Ok, I'll take that explanation. This project needs C++20, though, so there is a sweet spot somewhere in the middle.